src/main/java/ui/UiRenderer.java

```diff
@@ -0,0 +1,71 @@
+package ui;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+// uncomment below line when merging with Taskmanager
+// import main.TaskManager;
+
+public class UiRenderer {
+    private static final String[] WEEK_DAYS = {"Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"};
```

Comment:
We could perhaps use the built in capabilities of LocalDate package for the weekdays?